lib/matplotlib/quiver.py

```diff
@@ -72,12 +72,16 @@
 U, V : 1D or 2D array-like
     The x and y direction components of the arrow vectors.
 
+    They must have the same number of elements, matching the number of arrow
+    locations. They may be masked.
```

Comment:
With the same mask or is different allowed?

Reply:
They can be different.  The masks are combined so that a masked point in any of U, V, or C is the same as if all were masked.

Reply:
Maybe more precisely

> Elements that are masked in U or V are not drawn.

instead of just just

> They may be masked.

Reply:
I think commiting this suggested change will cause problems due to the line being too long for flake?